config/site_settings.yml

```diff
@@ -2061,10 +2058,12 @@ developer:
     allow_any: false
     refresh: true
 
-sidebar:
-  enable_sidebar:
```

Comment:
Note I'm not dropping any overrides in the DB in this PR yet since I want to be safe